apps/mobile/src/screens/NftSelectorScreen/NftSelectorPickerSingularAsset.tsx

```diff
@@ -56,6 +56,11 @@ export function NftSelectorPickerSingularAsset({
       setProfileImage(token.dbid).then(() => {
         onSelect();
       });
+    } else if (currentScreen === 'Community') {
```

Comment:
does this mean that if I'm on the community page , and then open the Nft Selector, and then post, I should stay on the Community Page?

I'm seeing that I get redirected to the Latest feed. just started a discussion in slack to agree on the correct redirect behavior!

Reply:
Yup! It should be redirect to community page


https://github.com/gallery-so/gallery/assets/4480258/b17fe27a-edc8-4383-bc24-ea100026e130

Reply:
@jakzaizzat i noticed after you post, it swipes back two screens, instead of going directly back to the community page in a single transition.

is there any way we could skip the intermediary screen and go directly back? probably a react navigation thing

maybe worth asking chatGPT!

Reply:
@Robinnnnn Thanks for the idea! Now the transition much smoother after the user post. It skipped the second screen


https://github.com/gallery-so/gallery/assets/4480258/46696dcd-2c95-451a-9786-b7709885c1c8